hack/requirements.txt

```diff
@@ -1 +1,2 @@
 PyGithub==1.52
+pytest==6.2.2
```

Comment:
I personally prefer sticking to the stdlib instead of using test-only deps, because it's one less thing we might forget to bump, and the stdlib support for testing is pretty good.